halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/profile/SpinnakerProfile.java

```diff
@@ -121,15 +131,37 @@ private ProfileConfig getBaseConfig(DeploymentConfiguration deploymentConfigurat
    * @param node is the node to find dependent files in.
    * @return the list of files required by the node to function.
    */
-  List<String> dependentFiles(Node node) {
+  List<String> processDependentFiles(Node node) {
```

Comment:
According to the method comment, this is processing files that are dependencies of this node (that is, files this node is dependent on); not files dependent on this node.